chrome-launcher/test/chrome-launcher-test.ts

```diff
@@ -102,4 +102,24 @@ describe('Launcher', () => {
     assert.strictEqual(pid, chromeInstance.pid);
     await chromeInstance.kill();
   });
+
+  it('launching flags without --disable-extensions', async () => {
+    const spawnStub = stub().returns({pid: 'some_pid'});
+
+    const chromeInstance = new Launcher(
+        {enableExtensions: true},
+        {fs: fsMock as any, rimraf: spy() as any, spawn: spawnStub as any});
+    stub(chromeInstance, 'waitUntilReady').returns(Promise.resolve());
+
+    chromeInstance.prepare();
+
+    try {
+      await chromeInstance.launch();
+    } catch (err) {
+      return Promise.reject(err);
+    }
+
+    const chromeFlags = spawnStub.getCall(0).args[1] as string[];
+    assert.ok(chromeFlags.indexOf('--disable-extensions') < 0)
```

Comment:
I've got an error, `test/chrome-launcher-test.ts (124,28): Property 'includes' does not exist on type 'string[]'. (2339)` To solve the error, we need to change target config in tsconfig.json from 'es6' to 'es2016'